marabunta/runner.py

```diff
@@ -28,8 +28,16 @@ def __init__(self, config, migration, database, table):
         # we don't want to do it again for another version
         self.upgraded_addons = set()
 
-    def log(self, message):
-        print_decorated(u'migration: {}'.format(message))
+    def log(self, message, decorated=True, stdout=True):
```

Comment:
I don't see any of these new options used. Am I wrong?

Reply:
As run of the backup command was moved from version to migration - we need those options in execution of the command here:
https://github.com/camptocamp/marabunta/blob/d949b0974d20a2f3d9e5df02aaef56d2769ea70a/marabunta/model.py#L342